website_canonical_url/tests/test_rendering.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Simone Orsi <simone.orsi@camptocamp.com>
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import HttpCase
+from odoo.modules.module import get_resource_path
+from odoo import tools
+from lxml.html import document_fromstring
+
+
+def load_xml(cr, module, filepath, kind='demo'):
+    tools.convert_file(
+        cr, module,
+        get_resource_path(module, filepath),
+        {}, mode='init', noupdate=False, kind=kind)
```

Comment:
This one is a very interesting pattern that could help resolving the eternal declare-in-test-case vs. declare-in-demo-data debate (ref. https://github.com/OCA/maintainer-tools/pull/213).

IMHO it should appear in OCA docs.

OTOH, I'm not sure we need a separate method that just calls another method and is used just once...